ColorPatterns/Scenes/Game/GameController.swift

```diff
@@ -42,23 +42,27 @@ final class GameController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         updateColorBars()
-        setupSwipeDownGesture(direction: .down)
         setupGameTimer()
         setupViewModelDelegate()
         setupScoreLabel()
     }
 
     @IBAction private func colorBarPressed(_ sender: UIButton) {
+        // TODO: Try not to use tags, make an another way to identify buttons
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/vandermesis/ColorPatterns/blob/c07cd9a3b7a63e61e86332aeb4fa4cabf8b81c17/ColorPatterns/Scenes/Game/GameController.swift#L51"></span>TODOs should be resolved (Try not to use tags, make an a…).<br />
<code>todo</code> <code>GameController.swift:51</code></td>
    </tr>
  </tbody>
</table>